src/js/api/config.js

```diff
@@ -39,7 +39,9 @@ define([
             liveBroadcast: 'Live broadcast',
             loadingAd: 'Loading ad',
             rewind: 'Rewind 10s',
-            nextup: 'Next Up'
+            nextuptext: 'Next Up',
```

Comment:
1) this should be camelCase like the rest of the props
2) Why include the text suffix? Nothing else does that - it's implicit